src/videojs-http-streaming.js

```diff
@@ -140,18 +141,52 @@ Hls.canPlaySource = function() {
     'your player\'s techOrder.');
 };
 
-const emeKeySystems = (keySystemOptions, videoPlaylist, audioPlaylist) => {
+const emeKeySystems = (keySystemOptions, mainSegmentLoader, audioSegmentLoader) => {
   if (!keySystemOptions) {
     return keySystemOptions;
   }
 
+  let videoMimeType;
+  let audioMimeType;
+
+  // if there is a mimeType associated with the audioSegmentLoader, then the audio
+  // and video mimeType and codec strings are already in the format we need to
+  // pass with the other key systems
+  if (audioSegmentLoader.mimeType_) {
+    videoMimeType = mainSegmentLoader.mimeType_;
```

Comment:
are these equivalent to `videoPlaylist.attributes.CODECS` and same for audio?

Reply:
The first bullet under "Specific Changes" addresses this a bit. The values of the codecs themselves are the same, but `mimeType_` is already formatted as `'type/subtype; codecs="codec1, codec2"'` while `attributes.CODECS` is only `'codec1, codec2'`

Reply:
ah, cool, forgot to refer back to that when re-reviewing.